Removes the debian-only --install-layout python-setuptools option

--- devscripts-2.15.7/scripts/Makefile~	2015-08-03 09:38:41.000000000 +0300
+++ devscripts-2.15.7/scripts/Makefile	2015-08-03 09:39:15.202484554 +0300
@@ -86,7 +86,7 @@
 	$(foreach python,$(shell py3versions -r ../debian/control),$(python) setup.py test$(\n))
 
 install: all
-	python3 setup.py install --root="$(DESTDIR)" --no-compile --install-layout=deb
+	python3 setup.py install --root="$(DESTDIR)" --no-compile
 	cp $(SCRIPTS) $(DESTDIR)$(BINDIR)
 	ln -sf edit-patch $(DESTDIR)$(BINDIR)/add-patch
 	install -d $(DESTDIR)$(COMPL_DIR)
