diff -urNp -x '*.orig' devscripts-2.18.6.org/scripts/Makefile devscripts-2.18.6/scripts/Makefile
--- devscripts-2.18.6.org/scripts/Makefile	2021-03-04 22:38:11.020414057 +0100
+++ devscripts-2.18.6/scripts/Makefile	2021-03-04 22:38:11.090413685 +0100
@@ -133,7 +133,7 @@ clean:
 test: test_pl test_sh test_py
 
 install: all
-	python3 setup.py install --root="$(DESTDIR)" --no-compile --install-layout=deb
+	python3 setup.py install --root="$(DESTDIR)" --no-compile
 	cp $(SCRIPTS) $(DESTDIR)$(BINDIR)
 	ln -sf edit-patch $(DESTDIR)$(BINDIR)/add-patch
 	install -d $(DESTDIR)$(COMPL_DIR)
