diff -urNp -x '*.orig' devscripts-2.18.6.org/po4a/Makefile devscripts-2.18.6/po4a/Makefile
--- devscripts-2.18.6.org/po4a/Makefile	2017-10-01 17:57:17.000000000 +0200
+++ devscripts-2.18.6/po4a/Makefile	2021-03-04 22:38:10.917081273 +0100
@@ -28,7 +28,7 @@ de/%.de.1: de/%.de.pod translate
 	pod2man --utf8 --center=" " --release="Debian-Hilfswerkzeuge" $< > $@
 de/%.de.1: de/%.de.dbk translate
 	xsltproc --nonet -o $@ \
-	  /usr/share/sgml/docbook/stylesheet/xsl/nwalsh/manpages/docbook.xsl $<
+	  /usr/share/sgml/docbook/xsl-stylesheets/manpages/docbook.xsl $<
 # xsltproc writes out to foo.1 even if you tell it to do otherwise, so we have
 # to rename the file after it's generated
 	mv de/$*.1 $@
@@ -41,7 +41,7 @@ fr/%.fr.1: fr/%.fr.pod translate
 	pod2man --utf8 --center=" " --release="Utilitaires Debian" $< > $@
 fr/%.fr.1: fr/%.fr.dbk translate
 	xsltproc --nonet -o $@ \
-	  /usr/share/sgml/docbook/stylesheet/xsl/nwalsh/manpages/docbook.xsl $<
+	  /usr/share/sgml/docbook/xsl-stylesheets/manpages/docbook.xsl $<
 # xsltproc writes out to foo.1 even if you tell it to do otherwise, so we have
 # to rename the file after it's generated
 	mv fr/$*.1 $@
diff -urNp -x '*.orig' devscripts-2.18.6.org/scripts/Makefile devscripts-2.18.6/scripts/Makefile
--- devscripts-2.18.6.org/scripts/Makefile	2018-09-06 21:20:10.000000000 +0200
+++ devscripts-2.18.6/scripts/Makefile	2021-03-04 22:38:10.917081273 +0100
@@ -86,7 +86,7 @@ endif
 	pod2man --utf8 --center=" " --release="Debian Utilities" $< > $@
 %.1: %.dbk
 	xsltproc --nonet -o $@ \
-	  /usr/share/sgml/docbook/stylesheet/xsl/nwalsh/manpages/docbook.xsl $<
+	  /usr/share/sgml/docbook/xsl-stylesheets/manpages/docbook.xsl $<
 
 # Syntax checker
 test_sh: $(SH_CHECKS)
